transformations/butter_fingers_perturbation_for_IL/key_mapping.json

```diff
@@ -0,0 +1 @@
, "\u09b8", "\u099c", "\u09b9", "\u09b0", "\u09b2"], "\u09a5": ["\u09a5", "\u09a7", "\u0996", "\u09b6", "\u099d", "\u0999", "\u09b1", "\u09b3"], "\u099a": ["\u099a", "\u09a4", ".", "\u09af", "\u099f", ","], "\u099b": ["\u099b", "\u09a5", "\u09e4", "\u09df", "\u09a0", "\u09b7"], "\u09d2": ["\u09d2", "\u09e5", "\u09f0"], "\u099f": ["\u099f", "\u09c9", "\u09bc", "\u09a1", "\u099a", "\u09af", "\u09a4", "."], "\u09a0": ["\u09a0", "\u0991", "\u099e", "\u09a2", "\u099b", "\u09df", "\u09a5", "\u09e4"], "\u09c9": ["\u09c9", "\u099f", "\u099a", "\u09af", "\u09bc", "\u09a1"], "\u0991": ["\u0991", "\u09a0", "\u099b", "\u09df", "\u099e", "\u09a2"], "\u09c6": ["\u09c6", "\u09cb", "\u0982", "\u09c7", "\u09a8", "\u09cd", "\u09be"], "\u098e": ["\u098e", "\u0993", "\u0981", "\u098f", "\u09a9", "\u0985", "\u0986"], "\u09d3": ["\u09d3"], "\u0982": ["\u0982", "\u09c6", "\u09ae", "\u09c7", "\u09cd", "\u09b5", "\u09a8", "\u09bf", "\u09c0", "\u09be", "\u09c8", "\u09cc"], "\u0981": ["\u0981", "\u098e", "\u09a3", "\u098f", "\u0985", "\u09b4", "\u09a9", "\u0987", "\u0988", "\u0986", "\u0990", "\u0994"], "\u09ae": ["\u09ae", "\u0982", "\u09a8", "\u09cd", "\u09bf", "\u09c6", "\u09c7", "\u09c8", "\u09be", "\u09c0", "\u09c1", "\u09b5"], "\u09a3": ["\u09a3", "\u0981", "\u09a9", "\u0985", "\u0987", "\u098e", "\u098f", "\u0990", "\u0986", "\u0988", "\u0989", "\u09b4"], "\u09d4": ["\u09d4", "\u09e2", "\u09d3", "\u09e3"], "\u09a8": ["\u09a8", "\u09ae", "\u09bf", "\u09b5", "\u09c1", "\u0982", "\u09cd", "\u09be", "\u09c0", "\u09c2", "\u09ac", "\u09b2"], "\u09a9": ["\u09a9", "\u09a3", "\u0987", "\u09b4", "\u0989", "\u0981", "\u0985", "\u0986", "\u0988", "\u098a", "\u09ad", "\u09b3"], "\u09b5": ["\u09b5", "\u09a8", "\u09b2", "\u09c1", "\u09aa", "\u09ae", "\u09bf", "\u09c2", "\u09ac", "\u09b9", "\u09b2"], "\u09b4": ["\u09b4", "\u09a9", "\u09b3", "\u0989", "\u09ab", "\u09a3", "\u0987", "\u098a", "\u09ad", "\u0999", "\u09b3"], "\u09b2": ["\u09b2", "\u09b5", "\u09b8", "\u09aa", "\u09b0", "\u09a8", "\u09c1", "\u09c2", "\u09b9", "\u0997", "\u0995"], "\u09b3": ["\u09b3", "\u09b4", "\u09b6", "\u09ab", "\u09b1", "\u09a9", "\u0989", "\u098a", "\u0999", "\u0998", "\u0996"], "\u09b8": ["\u09b8", "\u09b2", "\u0995", "\u09b0", "\u09a4", "\u09a6", "\u0997", "\u0995"], "\u09b6": ["\u09b6", "\u09b3", "\u0996", "\u09b1", "\u09a5", "\u09a7", "\u0998", "\u0996"], ",": [",", "\u09b8", ".", "\u09b2", "\u09b0", "\u0995", "\u09a4", "\u099a", "\u09af"], "\u09b7": ["\u09b7", "\u09b6", "\u09e4", "\u09b3", "\u09b1", "\u0996", "\u09a5", "\u099b", "\u09df"], "\u09f0": ["\u09f0", "\u09e5", "\u09d8", "\u09d2"], ".": [".", ",", "\u09a4", "\u099a", "\u09af", "\u09b8", "\u0995", "\u099f"], "\u09e4": ["\u09e4", "\u09b7", "\u09a5", "\u099b", "\u09df", "\u09b6", "\u0996", "\u09a0"], "\u09e5": ["\u09e5", "\u09f0", "\u09d2", "\u09d8"], "\u09af": ["\u09af", ".", "\u099a", "\u099f", "\u09c9", "\u0995", "\u09b8", "\u09a4"], "\u09df": ["\u09df", "\u09e4", "\u099b", "\u09a0", "\u0991", "\u0996", "\u09b6", "\u09a5"], " ": [" "]}}}
```

Comment:
Can you display them in non-UTF format for these characters to be visible?

Reply:
And also if you could format the JSON in a structured format, it will help readers edit the file manually too!

Reply:
I have indented the JSON and updated the encoding format. Let me know if it looks better.